ert_gui/simulation/models/base_run_model.py

```diff
@@ -248,17 +248,22 @@ def updateDetailedProgress(self):
             except ValueError:
                 continue
 
-            fms = self.realization_progress[iteration].get(run_arg.iens, None)
-            if fms and BaseRunModel.is_forward_model_finished(fms):
-                continue
+            status = None
+            if self._job_queue:
+                status = self._job_queue.getJobStatus(queue_index)
 
-            fms = ForwardModelStatus.load(run_arg.runpath, num_retry=1)
+            fms = self.realization_progress[iteration].get(run_arg.iens, None)
 
-            if not fms:
-                continue
+            #Dont load from file if you are finished
+            if fms and BaseRunModel.is_forward_model_finished(fms[0]):
+                jobs = self.realization_progress[iteration][run_arg.iens][0]
+            else:
+                fms = ForwardModelStatus.load(run_arg.runpath, num_retry=1)
+                if not fms:
+                    continue
 
-            jobs = fms.jobs
-            self.realization_progress[iteration][run_arg.iens] = jobs
+                jobs = fms.jobs
```

Comment:
May I suggest to no indent this line as `jobs` is only used outside this scope...

Reply:
it should either be loaded from buffer in memory or from file.
With no indent it will always try to use from file.

Reply:
You're right. Ignore this comment!